packages/jest-preset/transforms/styles.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  process() {
+    return 'module.exports = {};';
+  },
+  getCacheKey() {
+    return 'style-transition';
```

Comment:
"transition"?

Reply:
Yeah that's weird. I copied it from CRA. What does that even mean, I'll fix